sequence_processing_pipeline/Pipeline.py

```diff
@@ -12,11 +12,79 @@
 import sample_sheet
 import pandas as pd
 from collections import defaultdict
+from datetime import datetime
+import pytz
+from xml.etree import ElementTree as ET
 
 
 logging.basicConfig(format='%(asctime)s - %(message)s', level=logging.INFO)
 
 
+class InstrumentUtils():
+    types = {'A': 'NovaSeq 6000', 'D': 'HiSeq 2500', 'FS': 'iSeq',
+             'K': 'HiSeq 4000', 'LH': 'NovaSeq X Plus', 'M': 'MiSeq',
+             'MN': 'MiniSeq',
+             # SN – RapidRun which is HiSeq 2500
+             'SN': 'RapidRun'}
+
+    @staticmethod
```

Comment:
This was a hedge. I know that I naturally tend to resort to creating classes more than others in the lab do. In this case I did want to encapsulate the types within a scope, but I also demonstrated that none of the methods require state to be maintained in between them.

I wouldn't recommend making these regular methods for that reason. If anything I would recommend making types and the methods global instead of making them regular methods.

IMHO my first choice would be to migrate these to metapool, since they would be potentially useful in notebooks and other code. I can go either way on either keeping it a static class or making them global.